### DISCUSSION TITLE
Question and Answer Self-Host Site

### DISCUSSION THREAD

**Discussion by Cybersoulja** [Oneseco-Media/.github#32] [Ideas]
I think this site is a minimum effort set up that self hosted meaning we own it and it's free. We can use this site as a prototype for our social network.

@SapphireBaby would be the owner and I will be admin. This gives one co-founder her own piece of Oneseco Media and grows our reach without taking @Cybersoulja's focus away from his projects. 

- [ ] Requires @SapphireBaby approval before building starts
- [ ] Ownership must be transferred after setup to @SapphireBaby 
- [ ] If the co-founders vote down this project, this discussion and the project will be closed and archived.

**Comment by Cybersoulja** [comment]
Ugh, forgot to put the link again.

**Comment by SapphirePixie** [comment]
> I think this site is a minimum effort set up that self hosted meaning we own it and it's free. We can use this site as a prototype for our social network.
> 
> @SapphireBaby would be the owner and I will be admin. This gives one co-founder her own piece of Oneseco Media and grows our reach without taking @Cybersoulja's focus away from his projects.
> 
> * [ ]  Requires @SapphireBaby approval before building starts
> * [ ]  Ownership must be transferred after setup to @SapphireBaby
> * [ ]  If the co-founders vote down this project, this discussion and the project will be closed and archived.

>I'm on board!

**Reply by Cybersoulja** [reply]
[Memberpress Is Better](https://memberpress.com/) @SapphireBaby
